tests/test_spstats.py

```diff
@@ -18,11 +19,10 @@
     warnings.filterwarnings("ignore", category=DeprecationWarning)
     from skgstat import models
 
-
 PLOT = False
 
-
-def load_diff() -> tuple[gu.georaster.Raster, np.ndarray]:
+def load_diff() -> Tuple[gu.georaster.Raster, np.ndarray] :
```

Comment:
Why did you change this from `tuple` to `Tuple`? If a linter suggests it, the linter is outdated.

Reply:
I does this to me for `list` and `tuple`, and wants me to import `List` and `Tuple` from annotations instead. Are those two similarly outdated?

Reply:
Fixed